Formula/t/trezor-agent.rb

```diff
@@ -271,15 +265,26 @@ class TrezorAgent < Formula
   end
 
   resource "wheel" do
-    url "https://files.pythonhosted.org/packages/b8/d6/ac9cd92ea2ad502ff7c1ab683806a9deb34711a1e2bd8a59814e8fc27e69/wheel-0.43.0.tar.gz"
-    sha256 "465ef92c69fa5c5da2d1cf8ac40559a8c940886afcef87dcf14b9470862f1d85"
+    url "https://files.pythonhosted.org/packages/b7/a0/95e9e962c5fd9da11c1e28aa4c0d8210ab277b1ada951d2aee336b505813/wheel-0.44.0.tar.gz"
+    sha256 "a29c3f2817e95ab89aa4660681ad547c0e9547f20e75b0562fe7723c9a2a9d49"
   end
 
   def install
-    ENV.append "CFLAGS", "-I#{Formula["libusb"].include}/libusb-1.0"
+    # Help gcc to find libusb headers on Linux.
+    ENV.append "CFLAGS", "-I#{Formula["libusb"].opt_include}/libusb-1.0" unless OS.mac?
```

Comment:
No linkage to `libusb` on macOS, so maybe this is still needed everywhere?

Reply:
Newer `cython-hidapi` doesn't use `libusb` by default anymore[^1]. Though it will use a bundled `hidapi`[^2] unless corresponding environment variable is set and `pkg-config` is installed

[^1]: https://github.com/trezor/cython-hidapi/commit/5cae7c36f9cdcb9da792961e8ed78a64770401ea
[^2]: https://github.com/trezor/cython-hidapi/blob/0.14.0.post2/setup.py#L225-L232

Reply:
To add to above, from other resources, `libusb` dependency is still used via `libusb1` which finds it via dlopen (CDLL). So won't see linkage.

`cython-hidapi` did directly link to `libusb` so that may have been linked in an older bottle.